static/app/views/acceptProjectTransfer/index.tsx

```diff
@@ -47,7 +47,7 @@ class AcceptProjectTransfer extends DeprecatedAsyncView<Props, State> {
         if (!projectSlug) {
           window.location.href = `${sentryUrl}/organizations/${orgSlug}/projects/`;
         } else {
-          window.location.href = `${sentryUrl}/settings/${orgSlug}/projects/${projectSlug}/teams/`;
+          window.location.href = `${sentryUrl}/organizations/${orgSlug}/settings/${orgSlug}/projects/${projectSlug}/teams/`;
```

Comment:
One last thing! Have `orgSlug` in here twice now. If you remove that second one I think this will be good